internal/event/events.go

```diff
@@ -37,6 +37,6 @@ type PreviousApplicationLinked struct {
 type ReducedFeeRequested struct {
 	UID              string   `json:"uid"`
 	RequestType      string   `json:"requestType"`
-	Evidence         []string `json:"evidence"`
+	Evidence         []string `json:"evidence,omitempty"`
```

Comment:
https://github.com/ministryofjustice/opg-event-store/pull/15 when I wrote the Cypress test we were sending `null` here, which didn't match the spec either. So, first bug found by the new tool!